test/recipes/80-test_cms.t

```diff
@@ -1239,6 +1239,22 @@ with({ exit_checker => sub { return shift == 6; } },
             "Check failure during BIO setup with -stream is handled correctly");
     });
 
+# Check that kari encryption with originator does not segfault
+with({ exit_checker => sub { return shift == 3; } },
+    sub {
+        SKIP: {
+          skip "EC is not supported in this build", 1 if $no_ec;
+
+          ok(run(app(['openssl', 'cms', '-encrypt',
+                      '-in', srctop_file("test", "smcont.txt"), '-aes128',
+                      '-recip', catfile($smdir, "smec1.pem"),
+                      '-originator', catfile($smdir, "smec3-cert.pem"),
+                      '-inkey', catfile($smdir, "smec3-key.pem")
```

Comment:
```suggestion
                      '-originator', catfile($smdir, "smec3.pem"),
                      '-inkey', catfile($smdir, "smec3.pem")
```
No need to split the contents of `smec3.pem` into two extra redundant key/cert files.
Please do not add those. The app can simply use `smec3.pem` for both key and cert and will take either one as needed for the respective option (`-inkey` and `-originator`).